pandas/core/groupby/generic.py

```diff
@@ -1898,6 +1908,9 @@ def groupby_series(obj, col=None):
 
         if not self.as_index:
             results.index = ibase.default_index(len(results))
+            if results.ndim == 1:
```

Comment:
is this hit anywhere?

Reply:
No, it is not. obj_with_exclusions is always a frame, so I've removed the series path in this function.